returnn/tf/network.py

```diff
@@ -1054,6 +1051,8 @@ def add_layer(self, name, layer_class, **layer_desc):
           layer = subnet.net.add_layer(name=sub_name, layer_class=layer_class, **layer_desc)
       else:
         layer = self._create_layer(name=name, layer_class=layer_class, **layer_desc)
+    if name.__contains__("output") and name in self.layers:
```

Comment:
Why `__contains__`? Use standard Python code (`"output" in name`).

Reply:
oh you are right, I had smth. else in mind first I guess this was more of a relict of that.

Reply:
But also, this looks very heuristically. This doesn't really make sense.